lua/telescope/builtin/files.lua

```diff
@@ -53,19 +62,29 @@ end
 
 -- Special keys:
 --  opts.search -- the string to search.
+--  opts.search_dirs -- list of directory to search in
 files.grep_string = function(opts)
   -- TODO: This should probably check your visual selection as well, if you've got one
-  local search = opts.search or vim.fn.expand("<cword>")
-
-  search = escape_chars(search)
-
+  opts.vimgrep_arguments = opts.vimgrep_arguments or conf.vimgrep_arguments
+  opts.search = opts.search or vim.fn.expand("<cword>")
```

Comment:
Lets make both not part of opts. No real reason for that. Lets keep local. Same for `live_grep`

Reply:
done